client/asset/btc/btc.go

```diff
@@ -2573,6 +2573,12 @@ func (btc *ExchangeWallet) PayFee(address string, regFee, feeRateSuggestion uint
 	return newOutput(txHash, vout, sent), nil
 }
 
+// EstimateRegistrationTxFee gives a conservative enstimate for how much
+// transaction fees need to be spent to pay the registration fee.
+func (btc *ExchangeWallet) EstimateRegistrationTxFee() uint64 {
+	return (dexbtc.MinimumTxOverhead + (2 * dexbtc.P2PKHOutputSize) + dexbtc.RedeemP2PKHInputSize) * btc.feeRateLimit
+}
+
```

Comment:
Right now, 1400 sats/vB on a 140 vB tx would be `140 * 0.00001400 * 43500 = 85 USD` in tx fees for a registration tx, rather than the `0.36 USD` we might expect from an estimate using current network conditions.

Reply:
But they don't pay that.  Why does it matter the value of the over estimation here?

Reply:
Because if I tell the user that fees are 50 bucks, and then tell the user to deposit $135 bucks, and then only use $50.35, I look like a jerk and the user is confused and upset.

Reply:
OK, I guess, although people paying with asset X are almost certainly going to be trading asset X, so I don't see it going down like that.  But I take your point that showing a recommended deposit that is more than double the fee itself just looks stupid.
Still, I cannot stomach a fee rate request every time a WalletState is constructed, so as much as I think a new Core method (and accompanying api routes) is overkill, I don't see a way around that.